### PR TITLE
updating readme and adding docker-compose spec.yml

### PR DESCRIPTION
**Pull request by manovagyanik1**

### CODE REVIEW

**Comment by aametzler** [comment]
@manovagyanik1 Please add a description to your PR, so that I know what the description, purpose, steps to verify, expected behaviour are.

**Comment by manovagyanik1** [comment]
Adding the missing documents and required code changes that would help anyone to setup the code locally and run it


### Purpose

This would help the user in running the block exporter locally with setting up of mysql instance using docker and running the initial dump to create essential databases. Also removing the hardcoded database name from the SQL query. This would resolve 2 active issues

https://github.com/peerplays-network/Block-Exporter/issues/5
https://github.com/peerplays-network/Block-Exporter/issues/2

### Instruction to Verify
 ####
 1. Clone the fresh copy of the code
 2. Docker & docker-compose must be installed on the system, with the path /var/docker_volumes added to the file-sharing options of docker settings.
 3. nvm use 
 4. npm install
 5. docker-compose -f docker/docker-compose.dependency.yml up (up the mysql with dump script)
6. npm run stream


### Expected Behavior

As a developer, you should not be needing any other script, migrations or commands to run the code.

File: server/config/main.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /**
  * Store global settings (CONFIG)
  */
-const BLOCKCHAIN_URL = '';
+const BLOCKCHAIN_URL = 'wss://charlie.peerplays.download/api';
```
Comment (aametzler) [inline server/config/main.js]: Please remove. This is a testnet.

Comment (manovagyanik1) [reply server/config/main.js]: Removed

**Pull request merged**